plugins/modules/load_balancer_pool_member.py

```diff
@@ -0,0 +1,258 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright: (c) 2023, Gaudenz Steinlin <gaudenz.steinlin@cloudscale.ch>
+# Copyright: (c) 2023, Kenneth Joss <kenneth.joss@cloudscale.ch>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: load_balancer
+short_description: Manages load balancer pool members on the cloudscale.ch IaaS service
+description:
+  - List, create, update, delete pool members on the cloudscale.ch IaaS service.
+notes:
+  - If I(uuid) option is provided, it takes precedence over I(name) for pool member selection. This allows to update the load balancers's name.
+  - If no I(uuid) option is provided, I(name) is used for pool member selection. If more than one load balancer with this name exists, execution is aborted.
+author:
+  - Gaudenz Steinlin (@gaudenz)
+  - Kenneth Joss (@k-304)
+version_added: "2.3.0"
+options:
+  state:
+    description:
+      - State of the load balancer pool member.
+    choices: [ present, absent ]
+    default: present
+    type: str
+  name:
+    description:
+      - Name of the load balancer pool member.
+      - Either I(name) or I(uuid) are required.
+    type: str
+  uuid:
+    description:
+      - UUID of the load balancer.
+      - Either I(name) or I(uuid) are required.
+    type: str
+  load_balancer_pool:
+    description:
+      - UUID of the load balancer pool.
+    type: str
+  enabled:
+    description:
+      - Pool member will not receive traffic if false. Default is true.
+    default: true
+    type: bool
+  protocol_port:
+    description:
+      - The port to which actual traffic is sent.
+    type: str
+  monitor_port:
+    description:
+      - The port to which health monitor checks are sent.
+      - If not specified, protocol_port will be used. Default is null.
+    default: null
+    type: str
+  address:
+    description:
+      - The IP address to which traffic is sent.
+    type: str
+  subnet:
+    description:
+      - The subnet of the address must be specified here.
+    type: str
+  tags:
+    description:
+      - Tags assosiated with the load balancer. Set this to C({}) to clear any tags.
+    type: dict
+extends_documentation_fragment: cloudscale_ch.cloud.api_parameters
+'''
+
+EXAMPLES = '''
+# Create a pool member for a load balancer pool
```

Comment:
Like for the other examples I would add a complete example here without explicit UUIDs.

Reply:
Added such an example.